docker/test/stress/run.sh

```diff
@@ -97,6 +99,13 @@ EOL
     -->
     <core_path>$PWD</core_path>
 </clickhouse>
+EOL
+
+    # Let OOM killer terminate other processes before clickhouse-server:
+    cat > /etc/clickhouse-server/config.d/oom_score.xml <<EOL
+<clickhouse>
+    <oom_score>-1000</oom_score>
+</clickhouse>
```

Comment:
But we don't have "unneeded" processes while running stress tests in the CI. Which process will be killed if not `clickhouse-server`? `run.sh` or `dockerd`? It may lead to even more obscure issues.